### ISSUE TITLE
dashboard: ValueError: could not convert string to float: 'N/A'

### ISSUE THREAD

**Issue by jeromerobert** [jm-cc/gcvb#10]
I get this error when browsing to a failed test:
```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/robert/.local/lib/python3.7/site-packages/dash/dash.py", line 1461, in dispatch
    response.set_data(self.callback_map[output]["callback"](*args))
  File "/home/robert/.local/lib/python3.7/site-packages/dash/dash.py", line 1341, in add_context
    output_value = func(*args, **kwargs)  # %% callback invoked %%
  File "/home/robert/gcvb/gcvb/dashboard/apps/test.py", line 112, in display_page
    return gen_page(run,test_id)
  File "/home/robert/gcvb/gcvb/dashboard/apps/test.py", line 91, in gen_page
    data=data_preparation(run_id,test_id)
  File "/home/robert/gcvb/gcvb/dashboard/apps/test.py", line 41, in data_preparation
    elif float(v["distance"])>float(v["tolerance"]):
ValueError: could not convert string to float: 'N/A'

**Comment by jeromerobert** [comment]
This might be an unfinished test.

**Comment by jeromerobert** [comment]
Workaround:
```
cp -a gcvb/dashboard/assets /home/robert/miniconda3/lib/python3.8/site-packages/gcvb/dashboard/

**Comment by jm-cc** [comment]
Fixed in 186811e2e8e373df38e51c8f4fce990bb3767207
